sqe-database-access/Queries/EditionQueries.cs

```diff
@@ -105,7 +105,7 @@ FROM edition_editor
 
     # Get a thumbnail if possible
          LEFT JOIN (
-    SELECT iaa_edition_catalog.manuscript_id, MIN(CONCAT(proxy, url, SQE_image.filename)) AS thumbnail_url
+    SELECT iaa_edition_catalog.manuscript_id, MIN(CONCAT_WS('', proxy, url, SQE_image.filename)) AS thumbnail_url
```

Comment:
Why use CONCAT_WS here if the separator is an empty string? Any point in not using CONCAT?